### ISSUE TITLE
Error: Cannot find module 'axios/lib/utils'

### ISSUE THREAD

**Issue by kernoeb** [3846masa/axios-cookiejar-support#310]
Hello!

I recently updated the module from `0.4.2` to `1.0.0` in my **Electron** app.
Now, this error is showed after I build and start the app :

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'axios/lib/utils'
```

It is working when I'm in dev mode, the build is created, and it was also working before the update.
I think it's from the `require('axios/lib/utils')` in the code (?)

Any idea to help me ? 
Thanks :)

**Comment by azhaziqgoh** [comment]
@kernoeb , did you manage to overcome this? I face the same issue in my electron app.

**Comment by kernoeb** [comment]
> @kernoeb , did you manage to overcome this? I face the same issue in my electron app.

If I remember correctly, I finally used the old version of the module, unfortunately...

**Comment by 3846masa** [comment]
Sorry for the delay in response.
This issue has been resolved with the release of v2.0.0.

If you think this issue is still unresolved, please reopen it.
